kubetest/main.go

```diff
@@ -715,10 +715,15 @@ func chmodArtifacts() error {
 }
 
 func prepare(o *options) error {
+	k8sProviderEnvVar := o.provider
+	if o.provider == "kubernetes-anywhere" {
+		// To make gce-related setup scripts in k8s to get sourced.
+		k8sProviderEnvVar = "gce"
```

Comment:
That sounds wrong to me, but I'm lacking context here.

I think someone more familiar with this code should take a look.